storage/sql.go

```diff
@@ -1,9 +1,5 @@
 package storage
 
 func (conn *Connection) UpdateOnly(model interface{}, includeColumns ...string) error {
-	xcols, err := getExcludedColumns(model, includeColumns...)
-	if err != nil {
-		return err
-	}
-	return conn.Update(model, xcols...)
+	return conn.UpdateColumns(model, includeColumns...)
```

Comment:
do we always want the `updated_at` column to be updated automatically by pop? if so, we need to do the following as per the documentation:
```suggestion
        includeColumns = append(includeColumns, "updated_at")        
	return conn.UpdateColumns(model, includeColumns...)
```

if we don't then we need check where `UpdateOnly` is used and add the `updated_at` column to those that require the column to be updated